### ISSUE TITLE
Clarification of login/reg funnel data

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#13869]
I was looking at the data in the Top of the funnel, Registration funnel, and Login funnel and noticed that in all 3 dashboards the number of “engage” events is lower than the number of “submit” events. Hoping to understand what those specific events measure. 

[https://earthangel-b40313e5.influxcloud.net/d/RdmZ8ZL7z/all-fxa-funnels?orgId=1](https://earthangel-b40313e5.influxcloud.net/d/RdmZ8ZL7z/all-fxa-funnels?orgId=1)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-5672)

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

First, some definitions (source ( https://docs.telemetry.mozilla.org/datasets/fxa_metrics/funnels.html )):

  1. View means the page (or form, or whatever) has loaded and is displayed in the browser
  2. Engage means someone has clicked or focused one of the form inputs
  3. Submit means someone has attempted to submit the form.  It’s possible this fails, for example, if a required field is missing an error will be shown but this event will still be recorded.

And a caveat about front-end funnel events:

  1. There will always be some amount of inaccuracy in front-end event reporting.  These events (like viewing a form) happen in the client’s user agent.  That means we’re depending on that user agent to inform us of them.  Whether the events come back accurately are up to the user agent, the condition of the network in between, any add-ons the person is running, whether that person blocks certain requests, etc.  This inaccuracy could (and most likely will) manifest as missing data but it’s possible for someone to craft false metrics events also.

I have done some investigation on the top level fxa_email_first events.  Those come from the form on https://accounts.firefox.com ( https://accounts.firefox.com ) when you first visit and are not logged in.  We have around 30% more submit events than engage events for the past 2 days (but the trend continues for as long as we have data).

First I tried using the form and watching what events were sent.  I got the view, engage, and submit events sent as expected.  I didn’t get any anomalies in my testing.

Next I looked at the data and it also seems accurate.  I spot checked a half-dozen flows and they all looked normal.  

Then I came across a flow id with the normal view and engage events followed by four submit events.  Then the user appeared to register two separate accounts in under a minute, followed by 30 more submit events afterwards, some within two ten-thousandths of a second apart which seems like it is faster than someone would be able to interact with, even if they were holding the button down.  I assume this would either be a bug in the sending of the events or a bot sending events.

I don’t have an explanation for these extra events or why they would be so common as to change our actual trends so that submit is higher than engage.
